prime-react/src/main/scala/react/primereact/Dropdown.scala

```diff
@@ -18,12 +18,16 @@ case class Dropdown[A](
   options:         List[SelectItem[A]],
   id:              js.UndefOr[String] = js.undefined,
   clazz:           js.UndefOr[Css] = js.undefined,
+  panelClass:      js.UndefOr[Css] = js.undefined,
   filter:          js.UndefOr[Boolean] = js.undefined,
   showFilterClear: js.UndefOr[Boolean] = js.undefined,
   placeholder:     js.UndefOr[String] = js.undefined,
   disabled:        js.UndefOr[Boolean] = js.undefined,
   dropdownIcon:    js.UndefOr[String] = js.undefined,
+  tooltip:         js.UndefOr[String] = js.undefined,
+  tooltipOptions:  js.UndefOr[TooltipOptions] = js.undefined,
   onChange:        js.UndefOr[A => Callback] = js.undefined,
+  onChangeE:       js.UndefOr[ReactEvent => Callback] = js.undefined, // called after onChange
```

Comment:
So both callbacks are called instead of giving preference to one? I think in SUI we'd only trigger one.
What's the use case?

Reply:
The View based dropdowns make use of the `onChange` callbacks, so only the `onChangeE` is exposed by them. The use case was to do the `preventDefault` stuff, but it turned out I didn't need it. There may be other cases where access to the event might be required, but I can't think of any offhand. For the View based dropdowns, if you don't need the event, you can just `withOnMod`.

Reply:
My inability to come up with a use case made me question whether we should keep it.

Reply:
I guess I'd prefer to only expose one event rather than expose 2 where only one can really be used at a time. I thought that was kind of surprising behavior in SUI. But, for most of the components, the event is seldom used. If you'd prefer only one event, I can change it here and in the other controls. For the View based dropdowns I can chain the internal View changing handler with the passed in event handler.

But, even if both are kept, you're right the the `onChangeE` should be `(A, E) => Callback`.

Reply:
I suppose having both is fine as they are in an `UndefOr`, if you put two probably you should know what you're doing